libr/include/r_util/pj.h

```diff
@@ -39,6 +39,8 @@ R_API PJ *pj_kd(PJ *j, const char *k, double d);
 R_API PJ *pj_kf(PJ *j, const char *k, float d);
 R_API PJ *pj_kb(PJ *j, const char *k, bool v);
 R_API PJ *pj_null(PJ *j);
+R_API PJ *pj_r(PJ *j, const unsigned char *k, size_t len);
```

Comment:
`k` seems like a very misleading name, as it is usually used for `key`. However this is a value, so i suggest you rename it as `v`.

Reply:
and maybe `len` -> `v_len`